src/exercises/01-Query-Renderer/README.md

```diff
@@ -59,10 +83,96 @@ Let's update this page to show actual data about the current artist.
     </div>
   );
 ```
+_TODO: line numbers_
 
+Save the file, and you should see that we're not displaying anything useful on the artist detail screen: 
+
+![The artist detail screen showing no useful text](TODO)
+
+This is because our screen isn't loading any artist data.
 
 #### Add a QueryRenderer to retrieve artist data
 
+We're going to modify our Artist1QueryRenderer to use a Relay `<QueryRenderer>` component to connect our page to our GraphQL endpoint.
+
+💻 _Add an import statement for some dependencies to Artist1QueryRenderer:_
+
+```typescript
+import { graphql, QueryRenderer } from 'react-relay';
+import { environment } from '../../relay';
+```
+
+_TODO: line numbers_
+
+`QueryRenderer` is the component we're going to render; `graphql` helps us specify the GraphQL query that should be made.
```

Comment:
I wonder if it worth mentioning that the `QueryRenderer` is the root level of a query or relay tree. Calling out that in the Relay world it's the start of composing a query.

Reply:
Addressed in b37a8d2!